docs/src/main/docs/health/01_health.adoc

```diff
@@ -0,0 +1,133 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Health Checks
+:description: Helidon health checks
+:keywords: helidon, health-checks, health, check
+
+== Health Check Support
```

Comment:
General comment - for all section titles:

In a conversation with @tjquinno (in the context of the MP web service guide that he wrote), we agree to standardize our section headings to use the sentence case: Capitalize only the first letter of the title. Don't capitalize the first letter of the other words unless of course they happen to be proper nouns, like the name of a module.